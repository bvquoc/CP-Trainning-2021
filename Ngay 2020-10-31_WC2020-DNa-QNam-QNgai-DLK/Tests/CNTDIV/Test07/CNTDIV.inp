7
803
884
791
926
881
591
671
