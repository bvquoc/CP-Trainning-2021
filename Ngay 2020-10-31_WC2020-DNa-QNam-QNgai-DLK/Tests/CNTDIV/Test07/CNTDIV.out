5084
2296
3869
710
266
285
1145
